extensions/npm/package.nls.json

```diff
@@ -17,5 +18,6 @@
 	"command.debug": "Debug",
 	"command.openScript": "Open",
 	"command.runInstall": "Run Install",
-	"command.runSelectedScript": "Run Script"
+	"command.runSelectedScript": "Run Script",
+	"command.runScriptsFromFolder": "Select npm Scripts to Run..."
```

Comment:
Simplify the lable to "Run npm Script in Folder..." to make it similar to "Find in Folder...".